packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -27,7 +27,7 @@ var get = Ember.get;
   @class BuildURLMixin
   @namespace DS
 */
-var BuildURLMixin = Ember.Mixin.create({
+export default Ember.Mixin.create({
```

Comment:
That only works when the file is named appropriately (which seems fine in this specific case), often we have `ember-metal/utils` which is useless for fuzzy file finders.